### ISSUE TITLE
Big accessibility issue

### ISSUE THREAD

**Issue by Albert221** [CodiceApp/website#1]
At [this line](https://github.com/CodiceApp/website/blob/master/resources/scripts/demo.js#L31) and 2 more, there is a big accessibility issue. `.sr-only` is being used for hiding text because it is no longer valid in given context, but for screen readers it still remains and fools them. `display: none` should be used.
